Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -697,7 +759,7 @@ private void GenElementHandler(ActGenElement act)
                         if (e != null)
                         {
                             e.Click();
-                            DriverWindow.ShowActionEfect(true, 1000);
+                            if (DriverWindow != null) DriverWindow.ShowActionEfect(true, 1000);
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)